src/components/molecules/Schedule/Schedule.types.ts

```diff
@@ -0,0 +1,19 @@
+import { Room } from "types/rooms";
+import { PersonalizedVenueEvent } from "types/venues";
+import { WithVenueId } from "utils/id";
+
+export type RoomWithEvents = Room & {
+  events: WithVenueId<PersonalizedVenueEvent>[];
+};
```

Comment:
it should be moved to `types/rooms`

Reply:
moved to `types/rooms` https://github.com/sparkletown/sparkle/pull/1302/commits/97450fe8a6c8cff5135e9d09b572068adf69b71c